README.md

```diff
@@ -28,10 +28,26 @@ python3 setup.py install
 
 ## Quick Usage
 
-First, start the ape console using the network of your choice:
+The ENS plugin requires a mainnet connection to resolve ENS names because ENS contracts are only deployed to mainnet.
+Thus, the first thing you should do is ensure you have configured a mainnet provider.
+For example, if you use `infura` or `alchemy`, install the associated plugin:
 
```

Comment:
To check if you have the associated plugins you can try `ape plugins list`

Reply:
Addressed in 466ff1e!